plugins/catalog/src/api/CatalogClient.ts

```diff
@@ -45,14 +60,27 @@ export class CatalogClient implements CatalogApi {
     }
     return undefined;
   }
-  async getEntitiesByLocationId(id: string): Promise<Entity[]> {
-    const response = await fetch(
-      `${this.apiOrigin}${this.basePath}/entities?${LOCATION_ANNOTATION}=${id}`,
-    );
-    return await response.json();
-  }
-  async getEntities(): Promise<DescriptorEnvelope[]> {
-    const response = await fetch(`${this.apiOrigin}${this.basePath}/entities`);
+  async getEntities(
```

Comment:
hm is this something that you got by merging master into this or so? i think you will want to rebase your PR on master again and check that no unrelated changes get through

Reply:
I will